setup.py

```diff
@@ -27,7 +27,6 @@
     'eventlet',
     'gevent',
     'librabbitmq',
-    'lzma',
```

Comment:
backports.lzma; & 'lzma' same?

Reply:
An extra has a name. It can install multiple packages.